test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -30,14 +30,16 @@ import (
 )
 
 const (
-	WorkloadDir = "workloads"
-	PolicyDir   = "workloads/policies"
-	SSHPort     = "50001"
+	WorkloadDir           = "workloads"
+	PolicyDir             = "workloads/policies"
+	deleteResourceRetries = 10
```

Comment:
if we are going to use a const for number of retries and it's the same for every call, why pass it as a parameter to the function and not just use it in the delete function directly?

Reply:
Good question. Just an iterative placeholder until we have stronger opinions about particular use cases of delete that should be retried fewer or more times (or until we make it user-configurable)

Reply:
Thoughts on having a generic returnable function that takes in function/cmd parameters?